core/model/model_test.go

```diff
@@ -29,7 +29,27 @@ func (*ModelSuite) TestValidateSeries(c *gc.C) {
 		{model.CAAS, "bionic", false},
 		{model.CAAS, "kubernetes", true},
 	} {
-		err := model.ValidateSeries(t.modelType, t.series)
+		err := model.ValidateSeries(t.modelType, t.series, false)
+		if t.valid {
+			c.Check(err, jc.ErrorIsNil)
+		} else {
+			c.Check(err, jc.Satisfies, errors.IsNotValid)
+		}
+	}
+}
+
+func (*ModelSuite) TestValidateSeriesNewCharm(c *gc.C) {
+	for _, t := range []struct {
+		modelType model.ModelType
+		series    string
+		valid     bool
+	}{
+		{model.IAAS, "bionic", true},
+		{model.IAAS, "bionic", true},
+		{model.CAAS, "bionic", true},
```

Comment:
also include new examples with system etc

Reply:
the `series` here is the result parsed from `systems`. 
So the `systems` is not accessible from here(`ValidateSeries()`) anymore.